crates/vhost-device-gpio/src/backend.rs

```diff
@@ -43,10 +49,23 @@ pub(crate) enum Error {
     CouldNotCreateBackend(crate::vhu_gpio::Error),
     #[error("Could not create daemon: {0}")]
     CouldNotCreateDaemon(vhost_user_backend::Error),
+    #[cfg(not(feature = "mock_gpio"))]
+    #[error("Unimplemented")]
+    DeviceUnimplemented,
+}
+
+macro_rules! gpio_after_help {
+    () => {
+        concat!(
+            "Each device number here will be used to access the corresponding /dev/gpiochipX",
+            "or simulate a GPIO device with N pins (when feature enabled).",
```

Comment:
Are there no spaces missing here? I am also not sure why this needs to be a macro? Can't this be a usual (maybe const) function?

Reply:
It was a macro when we originally had two versions, I've made it a const string now.